packages/protoplugin-example/protoc-gen-twirp-es/package-lock.json

```diff
@@ -0,0 +1,100 @@
+{
+  "name": "protoc-gen-twirp-es",
+  "version": "0.1.0",
+  "lockfileVersion": 2,
```

Comment:
In this monorepo, there must only be a single package-lock.json file. 

We can't really ask users to run `make` (and have Go, Bazel, and all other tools installed), so it should be possible to check out the repository and run this example without. It's okay if the users needs to run `npm install` here, and that a new lock file is created.